src/Microsoft.AspNetCore.ResponseCaching/Internal/MemoryResponseCacheStore.cs

```diff
@@ -6,11 +6,11 @@
 
 namespace Microsoft.AspNetCore.ResponseCaching.Internal
 {
-    internal class MemoryResponseCache : IResponseCache
+    public class MemoryResponseCacheStore : IResponseCacheStore
```

Comment:
I thought about that too but it seems we call it `MemoryCache` everywhere. The type is called `MemoryCache`, the interface is `IMemoryCache` and most members are called `MemoryCache` https://github.com/aspnet/Mvc/blob/d4c7dc3a834edc507865816a59960c5b31f9b10c/src/Microsoft.AspNetCore.Mvc.TagHelpers/DistributedCacheTagHelper.cs#L47,
https://github.com/aspnet/EntityFramework/blob/e60330ef54e0ab02eb4010f91b252e2ed67177bd/src/Microsoft.EntityFrameworkCore/Internal/IDbContextServices.cs#L51,

Also I have not seen anything called `InMemoryCache` and I don't think that a pattern we want to start.